uclient-fetch.c

```diff
@@ -633,6 +670,26 @@ int main(int argc, char **argv)
 			case L_QUIET:
 				quiet = true;
 				break;
+			case L_HEADER:
+				tmp = strchr(optarg, ':');
+				if (!tmp)
+					return usage(progname);
+
+				*tmp++ = '\0';
+				while (isspace(*tmp))
+					++tmp;
+
+				if (*tmp == '\0' || !is_valid_header(optarg) || strchr(tmp, '\n'))
+					return usage(progname);
+
+				h = calloc(1, sizeof(*h));
```

Comment:
The calloc is for arrays, you can use just `malloc(sizeof(*h))`

Reply:
I kinda developed the habit to always use calloc as it also zeros the allocated memory. In this case it is true that this is unneeded because we are writing to pointers there right after.